Examples/Swift/ChangeWorldviewBoundariesExample.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+import Mapbox
+
+@objc(ChangeWorldviewBoundariesExample_Swift)
+
+class ChangeWorldviewBoundariesExample: UIViewController, MGLMapViewDelegate {
+
+    let mapView = MGLMapView()
+    let source = MGLVectorTileSource(identifier: "admin-bounds", configurationURL: NSURL(string: "mapbox://mapbox.mapbox-streets-v8")! as URL)
```

Comment:
If this source is already included in Mapbox Light by default, you can access it instead with `MGLStyle.source(identifier: String)`, which returns an optional `MGLSource`, instead of declaring a new source.